CMakeLists.txt

```
@@ -3055,6 +3057,29 @@ if(SDL_CCACHE)
   endif()
 endif()
 
+if (SDL_CLANG_TIDY)
```

Everything looks good, only some final touches.
```suggestion
if(SDL_CLANG_TIDY)
```